### PR TITLE
Fixing Vulnerabilities

### PR DESCRIPTION
**Pull request by shreyakapoor08**
## Proposed changes
All vulnerabilities fixed

## Types of changes

_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update (Documentation content changed)
- [ ] Other (please describe): 

## Checklist

_Put an `x` in the boxes that apply_

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

## Screenshots

Please attach the screenshots of the changes made in case of change in user interface
image

### CODE REVIEW

**Comment by praveenscience** [comment]
OMG you're the best! 😁

**Comment by shreyakapoor08** [comment]
> OMG you're the best! 😁 Is the app working fine?

Thankyou so much :') @praveenscience 
Yes the app is working fine
image

**Comment by praveenscience** [comment]
Typing something on the experience, the whole stuff collapses.

image

Clicking on Contact doesn't show anything.

image

**Comment by praveenscience** [comment]
The above might have not been caused by you but you said the application works fine?

**Comment by praveenscience** [comment]
@shreyakapoor08 It's because of #42 - everything broke. We need to now wait till @dibakar95 fixes it.

**Comment by shreyakapoor08** [comment]
> The above might have not been caused by you but you said the application works fine?

Ohh, I am so sorry, I just checked it by running the application. I did not click on any of the other tabs to check if it actually broke or not. I have checked now and it is collapsing as you said.

I am really sorry, from now on I will surely check the functioning of the whole application

**Comment by praveenscience** [comment]
@shreyakapoor08 Not your fault. Don't need to apologise. 😊

**Comment by shreyakapoor08** [comment]
> @shreyakapoor08 It's because of #42 - everything broke. We need to now wait till @dibakar95 fixes it.

Okay got it :)

**Comment by shreyakapoor08** [comment]
> @shreyakapoor08 Not your fault. Don't need to apologise. 😊

:) :)

**Comment by shreyakapoor08** [comment]
@praveenscience Are the issues resolve? Can this PR get merged now?

**Comment by shreyakapoor08** [comment]
Hey @praveenscience @vermakhushboo @PrathibhaS Any update on this PR ?

**Comment by praveenscience** [comment]
@shreyakapoor08 Not yet... The app is still in a broken state. Looking into it.

File: package.json [hunk 1]
```diff
@@ -4,11 +4,13 @@
   "private": true,
   "homepage": "https://app.rezume.best",
   "dependencies": {
+    "acron": "^1.0.5",
```
Comment (praveenscience) [inline package.json:7]: What does acron do here?

**Pull request closed without merging**